src/utils/csvParser/parser.ts

```diff
@@ -0,0 +1,595 @@
+import dayjs from 'dayjs'
+import dayjsCustomParseFormat from 'dayjs/plugin/customParseFormat'
+import dayjsUTC from 'dayjs/plugin/utc'
+import dayjsTimezone from 'dayjs/plugin/timezone'
+dayjs.extend(dayjsCustomParseFormat)
+dayjs.extend(dayjsUTC)
+dayjs.extend(dayjsTimezone)
```

Comment:
before you merge this in, let's pair up and see if we can do this without adding a new dependency

Reply:
yup! sorry, forgot we said we'd do that first